TSQL_Parser/TSQL_Parser/Clauses/Parsers/TSQLSubqueryHelper.cs

```diff
@@ -1,101 +1,96 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
 using TSQL.Expressions;
 using TSQL.Expressions.Parsers;
 using TSQL.Statements;
-using TSQL.Statements.Parsers;
 using TSQL.Tokens;
 
 namespace TSQL.Clauses.Parsers
 {
-	internal static class TSQLSubqueryHelper
```

Comment:
Was there any actual change in this file other than tabs to spaces?

Reply:
There shouldn't be a change to it.  But it looks like there could be.  I can grab an older version of the file and paste it in so that it isn't shown at all in the PR.